src/wooden-tree/defaultStore.js

```diff
@@ -0,0 +1,52 @@
+import Tree, { ActionTypes } from './';
+
+const actionMapper = {
+  [ActionTypes.EXPANDED]: Tree.nodeExpanded,
+  [ActionTypes.DISABLED]: Tree.nodeDisabled,
+  [ActionTypes.SELECTED]: Tree.nodeSelected,
+  [ActionTypes.CHILD_NODES]: Tree.nodeChildren,
+  [ActionTypes.LOADING]: Tree.nodeLoading,
+};
+
+export default (state = null, action) => {
+  if (!state) {
+    return state;
+  }
+
+  let node = null;
+  if (action.nodeId) {
+    node = Tree.nodeSelector(state, action.nodeId);
+    if (!node) {
+      return state;
+    }
+  }
+
+  switch (action.type) {
```

Comment:
If you add `CHECKED` and `ADD_NODES ` in the mapping you can remove this switch and just use
```JS
return Object.hasOwnProperty.call(actionMapper, action.type) ? 
  actionMapper[action.type](state, action, node) :
  state
```